huaweicloud/services/acceptance/common/resources.go

```diff
@@ -0,0 +1,79 @@
+package common
+
+import (
+	"fmt"
+
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/acceptance"
+)
+
+// TestSecGroup can be referred as `huaweicloud_networking_secgroup.test`
+func TestSecGroup(name string) string {
+	return fmt.Sprintf(`
+resource "huaweicloud_networking_secgroup" "test" {
+  name                 = "%s"
+  delete_default_rules = true
+}
+`, name)
+}
+
+// TestVpc can be referred as `huaweicloud_vpc.test` and `huaweicloud_vpc_subnet.test`
+func TestVpc(name string) string {
+	randCidr, randGatewayIp := acceptance.RandomCidrAndGatewayIp()
+	return fmt.Sprintf(`
+resource "huaweicloud_vpc" "test" {
+  name = "%[1]s"
+  cidr = "%[2]s"
+}
+
+resource "huaweicloud_vpc_subnet" "test" {
+  name       = "%[1]s"
+  vpc_id     = huaweicloud_vpc.test.id
+  cidr       = "%[2]s"
+  gateway_ip = "%[3]s"
+
+  primary_dns   = "100.125.1.250"
+  secondary_dns = "100.125.21.250"
```

Comment:
the DNS is unnecessary because the provider will add them according to different regions.

Reply:
ok